src/configure.ac

```diff
@@ -479,12 +479,18 @@ AS_IF(
                 [define if the libmodbus3 headers and library are available]
             )
             AC_SUBST(HAVE_LIBMODBUS3, yes)
+            # re-enable modbus related tests that might have been disabled by previous configure run
```

Comment:
I would be more specific to only delete the skip files that have been created by this script if we chose that option. So I would just add the skip file to the four tests that are affected even though this needs adaption if more mb2hal tests are added.